### ISSUE TITLE
go-dlv.el does not use go-dlv-command-name variable

### ISSUE THREAD

**Issue by liferooter** [benma/go-dlv.el#19]
# Additional information
**OS:** Arch Linux
**Emacs version:** 26.3
**go-dlv.el version:** 20191005.829
**Installed from:** MELPA via package.el
**Delve version:** 1.4.0
**Delve path:** ~/go/bin/dlv
**Is ~/go/bin/dlv in $PATH:** yes

# Steps to reproduce

1. Install Emacs
2. Install go-dlv.el from MELPA
3. Set go-dlv-command-name to "/home/gleb/go/bin/dlv"
4. Open file _~/go/src/gotanks/main.go_
5. M-x dlv RET RET

# Expected behavior
Emacs have to start GUD interface with Delve debuging _~/go/src/gotanks/main.go_

# Real behavior
Emacs opens new buffer with text "Current directory in /home/gleb/go/src/gotanks/main.go_" and prints "Searching for program: Нет такого файла или каталога, dlv" in echo area (I'm Russian, so my locale is Russian. Emacs isn't affected by my locale, but shell is affected. "Нет такого файла или каталога" means "There is no such file or directory".

**Comment by benma** [comment]
Does `dlv debug` work outside of emacs with your main.go?

Please try without any other Emacs modules loaded: `emacs -q`, then load only the go-dlv module and try again.

`Current directory in /home/gleb/go/src/gotanks/main.go_` looks weird. what does `ls -laR /home/gleb/go/src/gotanks/` say?

**Comment by liferooter** [comment]
Oops, sorry, I've found out what real problem is. Really, ~/go/src/gotanks/main,go can't be debugged correctly with Delve (it's Delve issue or result of using some very big libraries, it doesn't matters). But another file I've tested go-dlv with, simple Golang hello-world, showed me what is the real but less problem. go-dlv doesn't use my dlv path variable. I still can use it by set path just in executing, but I can't set it permanently. My go-dlv-command-name value is "/home/gleb/go/bin/dlv".
Sorry for my Russian English
